libc/cmake/modules/LLVMLibCCompileOptionRules.cmake

```diff
@@ -233,7 +233,7 @@ endfunction()
 
 function(_get_hermetic_test_compile_options output_var flags)
   _get_compile_options_from_flags(compile_flags ${flags})
-  list(APPEND compile_options ${LIBC_COMPILE_OPTIONS_DEFAULT} ${compile_flags}
+  set(compile_options ${LIBC_COMPILE_OPTIONS_DEFAULT} ${compile_flags}
```

Comment:
Isn't this just overwriting the flags from `_get_compile_options_from_flags`?

Reply:
Fixed.